apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.js

```diff
@@ -51,6 +53,8 @@ async function sendNotifications (newsItem) {
         return
     }
 
+    const cacheClient = getRedisClient('notifyResidentsAboutNewsItem', 'throttleCommonNewsItemsNotifications')
```

Comment:
Does this need to be done every time `sendNotifications` the function is called?
Can we take it outside?